patches/react-native-notifier+1.6.1.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/node_modules/react-native-notifier/src/Notifier.tsx b/node_modules/react-native-notifier/src/Notifier.tsx
+index 56c5819..4f31e78 100644
+--- a/node_modules/react-native-notifier/src/Notifier.tsx
++++ b/node_modules/react-native-notifier/src/Notifier.tsx
+@@ -44,6 +44,7 @@ export class NotifierRoot extends React.PureComponent<ShowNotificationParams, St
+       Component: NotificationComponent,
+       swipeEnabled: DEFAULT_SWIPE_ENABLED,
+       componentProps: {},
++      visible: false,
```

Comment:
I don't think there's a PR for their repo because this patch basically changes how their lib works.
Their lib basically keeps the push mounted at all times, it's just not visible.
In our case it is necessary to disassemble the push, mainly because of the audio.
And other state controls that are not possible to control because push is always mounted.

Reply:
Ok, but that's a _choice_.
You could very well do all that on top of a prop.

If it makes easier to manage the logic, I'm ok with it.
Just make sure the unmount is still animating.